<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="firstButton">
       <property name="text">
        <string>Первое</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="soup1">
       <property name="text">
        <string>Суп Харчо</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="soup2">
       <property name="text">
        <string>Том Ням</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="soup3">
       <property name="text">
        <string>Борщ Славянский</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>91</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="secondButton">
       <property name="text">
        <string>Второе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="second1">
       <property name="text">
        <string>Котлетка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="second2">
       <property name="text">
        <string>Пюрешка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="second3">
       <property name="text">
        <string>Макарони</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="saladButton">
       <property name="text">
        <string>Салат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="salad1">
       <property name="text">
        <string>Цезарь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="salad2">
       <property name="text">
        <string>Дворянский</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="salad3">
       <property name="text">
        <string>Витаминный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>91</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="drinkButton">
       <property name="text">
        <string>Напиток</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="drink1">
       <property name="text">
        <string>Чай</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="drink2">
       <property name="text">
        <string>Кофе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="drink3">
       <property name="text">
        <string>Компот</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>91</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QCheckBox" name="desertButton">
       <property name="text">
        <string>Десерт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="desert1">
       <property name="text">
        <string>Тирамису</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="desert2">
       <property name="text">
        <string>Наполеон</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color: rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="orderButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Заказать</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
